sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- [[#4784]](https://github.com/Azure/azure-sdk-for-cpp/issues/4784) The `WIL::WIL` CMake target is no longer defined with `find_package(azure-core-cpp)`. To use WIL you have to write `find_package(WIL)`.
```

Comment:
I am not sure if we want this to be the user experience.
I am blocking this for now, so that it does not get merged by mistake, but I am open to discussion.

As I understood, you need this for the package to work in Conda. Are there alternative ways? What is the problem of having this as a dependency, just as we do for libcurl? Why don't vcpkg users have this problem? In the edge case, if this is what is really needed, can you apply patch in Conda (kind of how it is done in vcpkg - https://github.com/microsoft/vcpkg/pull/32425/commits/cffa6925adc0e45190e52c92c4a1dd9eb019792b)

We can continue the conversation here, or in the issue you opened.

Reply:
@antkmsft Thanks for the feedback! We went ahead with your suggestion and [applied a patch](https://github.com/conda-forge/staged-recipes/pull/23297/commits/dc448f7cb1bdfecdd3154977e8994ec94750d409) to our conda build